science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+<thead>
+<tr>
+<th style="text-align:left;">
+Kennzahl
+</th>
+<th style="text-align:left;">
+Definition
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Erstmalige Downloads
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Gesamtzahl der erstmaligen Downloads der App.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Erneute Downloads
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Gesamtzahl der erneuten Downloads der Anwendung. Enthält keine automatischen Aktualisierungen oder Wiederherstellungen von Geräten.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Downloads gesamt
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Gesamtzahl der App-Downloads, einschließlich erstmaliger Downloads und erneuter Downloads.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Installationen
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Gesamtzahl der Installationen der App auf Geräten mit iOS 8 oder tvOS 9 oder höher. Wiederholte Downloads auf demselben Gerät, Downloads auf mehrere Geräte mit derselben Apple ID und Installationen mit Familienfreigabe sind eingeschlossen. Die Gesamtzahlen basieren auf App-Nutzenden, die zustimmen, ihre Daten zu teilen.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Sitzung
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Anzahl, wie oft die App mindestens zwei Sekunden lang verwendet wurde. Wenn sich die App im Hintergrund befindet und später erneut verwendet wird, zählt dies als weitere Sitzung. Die Gesamtzahlen basieren auf App-Nutzenden, die zustimmen, ihre Daten zu teilen.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Aktive Geräte
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Anzahl der Geräte mit mindestens einer Sitzung während des ausgewählten Zeitraums. Basierend auf Geräten mit iOS 8 oder tvOS 9 oder höher. Die Gesamtzahlen basieren auf App-Nutzende, die zustimmen, ihre Daten zu teilen.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Aktiv in den letzten 30 Tagen
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Anzahl der aktiven Geräte mit mindestens einer Sitzung in den letzten 30 Tagen. Die Gesamtzahlen basieren auf App-Nutzenden, die eingewilligt haben, ihre Daten zu teilen.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+Abstürze
+</td>
+<td style="text-align:left;width: 11cm; ">
+Die Gesamtzahl der Abstürze auf Geräten mit iOS 8 oder tvOS 9 oder höher.
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+-keine Angabe-
+</td>
+<td style="text-align:left;width: 11cm; ">
+Holen Sie sich detaillierte Absturzprotokolle und Absturzberichte in Xcode, z. B. eindeutige Gesamtzahlen für jede Art von Absturz und die Anzahl der Benutzenden, die davon betroffen waren. Weitere Informationen findet man unter „Absturzberichte in der Xcode-Hilfe anzeigen“.
```

Comment:
I could not make sense of this entry.

Reply:
Yes, this line can be deleted. @dsarkar has already been informed.

Reply:
Fixed with https://github.com/corona-warn-app/cwa-website/pull/2501